mint-auditor/src/http_api/launch_rocket.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Function for starting the HTTP server
+
+use crate::{
+    db::MintAuditorDb,
+    http_api::{routes, service::MintAuditorHttpService},
+};
+use rocket::{custom, routes, Config};
+use std::{net::Ipv4Addr, str::FromStr};
+
+/// Start the http server
+pub async fn start_http_server(db: MintAuditorDb, port: Option<u16>, host: Option<String>) {
```

Comment:
I think this can just live in `mod.rs` but if not, it should be re-exported there. We try to not expose the internal hierarchy to the callers/users.